man/systemd.network.xml

```diff
@@ -2062,6 +2097,13 @@ Table=1234</programlisting></para>
           <ulink url="https://tools.ietf.org/html/rfc5227">RFC 5227</ulink>. Defaults to false.</para>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetLabel=</varname></term>
+        <listitem>
+          <para>See <varname>NetLabel=</varname> in [Address] section.</para>
```

Comment:
referencing the [Address] section is good, but please explicitly say that this option here applies labels to the DHCP acquired IP addresses the same way as NetLabel in [Address] applies them to statically configured ones.

Reply:
Now more explicit